opm/input/eclipse/Parser/Parser.hpp

```diff
@@ -97,7 +97,30 @@ namespace Opm {
         bool hasKeyword( const std::string& ) const;
         const ParserKeyword& getKeyword(const std::string& name) const;
 
-        bool isRecognizedKeyword( const std::string_view& deckKeywordName) const;
+        /// Whether or not string is a valid keyword.
+        ///
+        /// The full keyword recognition process first verifies that the
+        /// input string looks like a valid keyword and, if so matches that
+        /// string aginst the builtin set of know keywords.  The function
```

Comment:
against

Reply:
> against, known

Good catch.  I've fixed the documentation now.